docs/copilot/advanced.md

```diff
@@ -0,0 +1,84 @@
+---
+Order: 3
+Area: copilot
+TOCTitle: Advanced
+ContentId: c77dcce9-4ba9-40ac-8ae5-2df855088090
+PageTitle: GitHub Copilot advanced usage
+DateApproved: 12/7/2023
+MetaDescription: Enhance your coding with AI-powered suggestions from GitHub Copilot in Visual Studio Code.
+---
+# GitHub Copilot advanced
+
+GitHub Copilot has several advanced features and settings, such as agents and slash commands. This article explains these advanced features in detail.
+
+If you're new to VS Code or GitHub Copilot, you might want to review the [GitHub Copilot Overview](/docs/copilot/overview.md) article first.
+
+## Agents and slash commands
+
+To further help Copilot give you more relevant answers, you can indicate the scope and intent of your question through **agents** and **slash commands**.
+
+Agents are like experts who have a specialty that they can help you with, and you can talk to them in the chat by mentioning them with the `@` symbol. Currently, there are the following agents:
+
+* `@workspace` has context about the code in your workspace and can help you navigate it, finding relevant files or classes.
+* `@vscode` knows about commands and features in the VS Code editor itself, and can help you use them.
+* `@terminal` has context about the integrated terminal shell and its contents.
+
+You can prepend your chat inputs with a specific agent to help Copilot give you a more relevant response.
+
+![Asking the @vscode agent how to change the VS Code colors](images/advanced/agent-example.png)
+
+Agents can also support slash commands for specific types of questions or tasks. For example, `@workspace /explain` can be used to ask Copilot to explain a file or code selection in the context of your open workspace. To see the available agents and slash commands, type `/` for a list.
+
+![Copilot slash command list in chat](images/advanced/slash-commands.png)
+
+* @workspace /explain: Explain step-by-step how the selected code works.
+* @workspace /fix: Propose a fix for the bugs in the selected code.
+* @workspace /new: Create a new project based on a natural language description.
+* @workspace /newNotebook: Create a new Jupyter Notebook based on your description.
+* @workspace /tests: Generate unit tests for the selected code.
+* @vscode /api: Questions about VS Code extension development.
+* @vscode /search: Generate query parameters for workspace search.
+* @terminal: Explain how to do something in the integrated terminal.
+
```

Comment:
Should we add a section about suggested fixes for failed terminal commands? We can then refer from the terminal docs: https://github.com/ntrogh/vscode-docs/blob/c110423efc2b474d97bbc2c242cfc101e27d244a/docs/terminal/shell-integration.md#L115-L118

Reply:
I think we can have a terminal section, and start small with suggested fixes.
We might add more terminal copilot integration in the future, so it is good to have an open-ended section.

fyi @Tyriar @meganrogge